packages/workflow-diagram/test/layout.test.ts

```diff
@@ -0,0 +1,50 @@
+import test from 'ava';
+
+import { ProjectSpace } from '../dist/types';
+import { toElkNode, toFlow, doLayout } from '../src/layout/index';
+import { FlowElkNode, FlowNodeEdges } from '../src/layout/types';
+import { getFixture } from './helpers';
+
+test.skip('toElkNode should convert a project space to a workflow', async (t) => {
```

Comment:
I've had to skip this test because the returned tree doesn't have any children, which is a bit worrying.

This will need to be resolved before merging.